javascript/ql/src/Security/CWE-312/CleartextLogging.ql

```diff
@@ -38,5 +38,5 @@ where
   cfg.hasFlowPath(source, sink) and
   // ignore logging to the browser console (even though it is not a good practice)
   not inBrowserEnvironment(sink.getNode().asExpr().getTopLevel())
-select sink.getNode(), source, sink, "Sensitive data returned by $@ is logged here.",
-  source.getNode(), source.getNode().(Source).describe()
+select sink.getNode(), source, sink, "$@ is logged here.", source.getNode(),
```

Comment:
Is this a place where "this location" was unsuitable?

Reply:
I was trying to avoid "this location" in general.

Reply:
Well, yes, I also tried that.  
I tried to avoid "here" and "this location" that doesn't describe what is happening, and use "Sensitive data", "command execution", or something else that describes what is happening.  
But for this alert-message I didn't find a good message that avoided the "here".